CanvasColorSpaceProposal.md

```diff
@@ -125,8 +120,12 @@ In implementations in which a ``UNPACK_COLORSPACE_CONVERSION_WEBGL`` of ``BROWSE
 
 Note that, while convenient, this behavior is likely less efficient than specifying color conversion in ``ImageBitmapOptions``, where the color conversion may be done asynchronously and simultaneously with image decode.
 
+An mechanism for using ``ImageBitmapOptions`` to specify color space conversion will be added in the context of WebGL color space support, to support asynchronous and decode-time color space conversion for texture data upload.
```

Comment:
An mechanism -> A mechanism

WebGL's spec is managed outside of the W3C, so it's infeasible for it to make changes to the ImageBitmapOptions dictionary. Those changes will need to be made in the WHATWG spec.